### DISCUSSION TITLE
Max iOS audio recording length?

### DISCUSSION THREAD

**Discussion by CristianAThompson** [expo/expo#13788] [Archived: Questions]
Additional context, we use expo-av recorder to make audio recordings on iOS devices only at the moment and have noticed there is a hard cap at 25 minutes and 21 seconds on iOS regardless of iOS device. The recorder acts like it's still recording but only up until that duration is there actual audio capture. Anything after that time stamp consistenly has no actual capture. I know you can specify a max file size on android but is there some hidden mechanism that enforces a max file size/ audio duration on iOS?

**Comment by JCcastagne** [comment]
Bumping this^

Still a question I'd like to have answered!

**Reply by zcharef** [reply]
same :)

**Reply by tsapeta** [reply]
Hey @zcharef, just to confirm, are you using `expo-audio` or `expo-av` (which is now deprecated)?

**Reply by zcharef** [reply]
> Hey @zcharef, just to confirm, are you using `expo-audio` or `expo-av` (which is now deprecated)?

hey, I am using expo-audio
